crates/red_knot_python_semantic/resources/mdtest/narrow/type.md

```diff
@@ -99,9 +99,9 @@ def _(x: str | int):
 class A: ...
 class B: ...
 
-alias_for_type = type
```

Comment:
I can not annotate this alias as `Literal[type]`, as that can't be spelled (except using `knot_extensions.TypeOf`, which I didn't want to use here).